internal/service/sharing_tf/legacy_model.go

```diff
@@ -3703,7 +3703,7 @@ func (c ShareInfo_SdkV2) ApplySchemaCustomizations(attrs map[string]tfschema.Att
 	attrs["created_at"] = attrs["created_at"].SetComputed()
 	attrs["created_by"] = attrs["created_by"].SetComputed()
 	attrs["name"] = attrs["name"].SetOptional()
-	attrs["object"] = attrs["object"].SetOptional()
+	attrs["object"] = attrs["object"].SetComputed()
```

Comment:
This needs to remain optional. (universe PR is pending)

```suggestion
	attrs["object"] = attrs["object"].SetOptional()
```